app/views/catalog/_homepage_banner.html.erb

```diff
@@ -17,7 +17,22 @@
     </div>
 
 <% end %>
+
 <% if controller.controller_name == 'catalog' && @document && @document[:has_model_ssim] && @document[:has_model_ssim][0] == 'Collection' %>
+
+<!--<div style="display:none">
+  <%#= @pagination_collection.total_count %>
+</div>
+<div style="display:none">
+  <%#= @display_facet.items %>
+</div>
+<div style="display:none;">
+  <%#= @pagination_collection.items %>
+</div>
+<div style="display:none;">
+  <%#= @response_collection %>
+</div> -->
+
```

Comment:
Are these supposed to be commented out?

Reply:
Yes this needs to stay for debugging later.